jvb/src/main/java/org/jitsi/videobridge/Conference.java

```diff
@@ -763,6 +763,21 @@ public Relay getRelay(@NotNull String id)
         return relaysById.get(id);
     }
 
+    @Nullable
+    public AbstractEndpoint findSourceOwner(@NotNull String sourceName)
+    {
+        // TODO does it need to also handle OctoEndpoints?
```

Comment:
Yes, this needs to work with all endpoints. With Octo/Relayed endpoints we calculate the local max and then request that from the remote bridge.

Reply:
I've adjusted it to use `endpointsById` which should include Octo endpoints as well